inference-engine/src/mkldnn_plugin/nodes/mkldnn_matmul_node.cpp

```diff
@@ -70,103 +106,135 @@ void MKLDNNMatMulNode::getSupportedDescriptors() {
     if (getChildEdges().empty())
         IE_THROW()  << errorPrefix << " has incorrect number of output edges for layer " << getName();
 
-    auto inDims0 = getParentEdgeAt(0)->getShape().getStaticDims();
-    auto inDims1 = getParentEdgeAt(1)->getShape().getStaticDims();
-    auto outDims = getChildEdgeAt(0)->getShape().getStaticDims();
+    auto firstInPortPrec = getOriginalInputPrecisionAtPort(0);
+    auto secondInPortPrec = getOriginalInputPrecisionAtPort(1);
+
+    firstInPortPrec = secondInPortPrec = getMaxPrecision(getOriginalInputPrecisions());
```

Comment:
using getMaxPrecision here might fail case then firstInPortPrec == u8 and secondInPortPrec == s8. It will change secondInPortPrec to be equal u8

Reply:
Corrected